### ISSUE TITLE
[Bug]: Constant crashes while searching in JEI

### ISSUE THREAD

**Issue by interneth3ro** [FTBTeam/FTB-Modpack-Issues#373]
### Modpack

FTB Oceanblock

### Modpack version

1.9.0

### Log Files

[2021-11-25-1.log.gz](https://github.com/FTBTeam/FTB-Modpack-Issues/files/7604181/2021-11-25-1.log.gz)

### Describe the bug

I am getting consistent pack crashes while searching in JEI with launcher reporting error code 0. Not specific to any mod/item that I can see. I apparently do not have a crash_reports folder for my instance, so I attached my latest log. I really wish I had more specific information (reproduction steps, etc.) for you, as I am a software engineer myself and understand how important they are, but this is all I have at this point. I will update with a crash report if I can generate one.

### Steps to reproduce

Basically just crashes to desktop while searching in JEI. No specific reproduction steps that I can find.

### Expected behaviour

Ummm... shouldn't crash?

**Comment by Raidobw2** [comment]
See #40. It's a Java crash, therefore the log is not in the game folders, but directly inside the modpack folder. They look like hs_err_pid.
Open one and look where it crashed, I think from memory It's atio6axx.dll or something like that, basically it's your amd graphics card driver for open gl. If you search for the term "draconic" in that crash you'll see the stacktrace and the mod draconic evolution causing it.

Fix is in the issue 40 I linked. Cheers.

**Comment by interneth3ro** [comment]
Away from my computer, but will check that linked issue soon. Thanks for
the response! :)
